<launch>
    <rosparam file="$(find ros_tcp_endpoint)/config/params.yaml" command="load"/>
    <node name="server_endpoint" pkg="ros_tcp_endpoint" type="default_server_endpoint.py" args="--wait" output="screen" respawn="true" />
    <!-- <node pkg="unityrender" type="CFS.x86_64" name="unityrender" /> -->

    <node name="rviz" pkg="rviz" type="rviz" output="log" args="-d $(find ros_tcp_endpoint)/config/pcl.rviz" />
</launch>
